assets/css/tailwind.css

```diff
@@ -1,3 +1,9 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer components {
+    .btn {
+        @apply bg-[#E5CB86] text-[#E95829] p-5 rounded-md text-sm;
```

Comment:
use our colors like bg-pomelo-red for #e95829 bg-pomelo-yellow for #e5cb86 

in your dev environment you can see [here](http://localhost:3000/_tailwind/) all active colors etc.